python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -5415,9 +5415,8 @@ tables:
     nullable: false
     ivoa:ucd: meta.id;obs.image
   - name: detector
-    description: Id of the detector where this source was measured. Datatype short
-      instead of byte because of DB concerns about unsigned bytes.
-    datatype: long
+    description: Id of the detector where this source was measured.
+    datatype: int
```

Comment:
Did you actually mean to use `short` here and not `int`? Or is that not supported properly in Firefly?

Reply:
See @iagaponenko's reply on #dm-db-team - for these two tables the physical type in Qserv is "32-bit integer", for which Felis `int` is the correct match; for the others, `long` is still correct because the physical type is "64-bit integer".

We will talk to AP and DRP about whether to genuinely use `short` / `int16` in the upstream Parquet-file generation in the future.